patches/modules/sdc_devel/remove-boolean-filter-default.patch

```diff
@@ -0,0 +1,54 @@
+diff --git a/src/Plugin/TwigValidatorRule/TwigValidatorRuleFilter.php b/src/Plugin/TwigValidatorRule/TwigValidatorRuleFilter.php
+index 4b4b68d..8709bff 100644
+--- a/src/Plugin/TwigValidatorRule/TwigValidatorRuleFilter.php
++++ b/src/Plugin/TwigValidatorRule/TwigValidatorRuleFilter.php
+@@ -236,37 +236,6 @@ final class TwigValidatorRuleFilter extends TwigValidatorRulePluginBase {
+    */
+   private static function default(string $id, Node $node, Node $parent, array $definition): array {
+     $errors = [];
+-
+-    if (\is_a($parent, 'Twig\Node\Expression\ConstantExpression') && $parent->hasAttribute('value')) {
+-      $value = $parent->getAttribute('value');
+-      if (\is_bool($value) || NULL === $value) {
+-        $errors[] = ValidatorMessage::createForNode($id, $node, new TranslatableMarkup('Filter `default` is not for booleans or null!'));
+-      }
+-    }
+-
+-    // Detect {{ foo|default(bar) }} where bar is boolean case.
```

Comment:
This is for detecting boolean with default filter. Happy to keep it in if you want to update it, I think there were a _lot_ of errors for this.

Personally I like being type specific with my variables, i dont know why best practice is to leave it as a 'falsy' value